### DISCUSSION TITLE
Changing pl-PL.yml lang file to new one

### DISCUSSION THREAD

**Discussion by QuestionCatcher** [BG-Software-LLC/SuperiorSkyblock2#1746] [Pending Feature Requests]
Hi o/
I saw a few mistranslations in polish language relative to English so i fixed it.
I would appreciate changing the old file on new one for future Devs 
[pl.PL.txt](https://github.com/BG-Software-LLC/SuperiorSkyblock2/files/11760567/pl.PL.txt)

**Comment by OmerBenGera** [comment]
I went through the file and sensed an issue.
You translated `-cancel` to your own language, however this doesn't make sense as `-cancel` is what the plugin detects when typed.
You should not translate command names or any other such things that the plugin is hardcoded to look for.
